### DISCUSSION TITLE
[Feature] Add multi select to folders view

### DISCUSSION THREAD

**Discussion by pearcemichal** [immich-app/immich#14132] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would like to be able to select multiple images while in the folders browser, in order to preform mass actions on these images, such as adding tags or changing locations. It would also be nice if there were also a "select all" option for selecting all images in a folder. 

In my case, I am using external libraries to load in my images, which are arranged in a detailed subfolder structure that makes it easy to identify different content types, so being able to perform mass actions on images on a per folder basis would make tagging much easier. 

Thank you!

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#9159

**Reply by nielstron** [reply]
now reopened in #14307
